tests/conftest.py

```diff
@@ -208,3 +208,11 @@ def netrc_contents(
     monkeypatch.setenv("NETRC", str(netrc_file_path))
 
     return netrc_file_path
+
+
+@pytest.fixture
+def start_connection():
+    with mock.patch(
+        "aiohttp.connector.aiohappyeyeballs.start_connection"
```

Comment:
If possible, it would be preferred to use `autospec=True, spec_set=True`, to minimise the possibility of misusing the mock.

Reply:
349203019c80ce4af4e3514eff3e8976c4e485a5